packages/backend-test-utils/src/next/wiring/ServiceFactoryTester.ts

```diff
@@ -104,7 +104,8 @@ export class ServiceFactoryTester<TService, TScope extends 'root' | 'plugin'> {
     ...args: 'root' extends TScope ? [] : [pluginId?: string]
   ): Promise<TService> {
     const [pluginId] = args;
-    return this.#registry.get(this.#subject, pluginId ?? 'test')!;
+    const instance = this.#registry.get(this.#subject, pluginId ?? 'test')!;
+    return Array.isArray(instance) ? instance[0] : instance;
```

Comment:
As discussed we'll want to forward all instances here and below

Reply:
Fixed 🙌🏻